From: Bernd Zeimetz <bernd@bzed.de>
Date: Thu, 28 Jul 2011 11:01:21 +0200
Subject: Fix the plugin directory path.

As plugins are shared libraries, they need to go into /usr/lib,
not into /usr/share.

Thanks: Andrew McMillan
---
 librawstudio/Makefile.am                 |    1 +
 librawstudio/rawstudio-2.0.pc.in         |    2 +-
 librawstudio/rs-plugin-manager.c         |    2 +-
 plugins/cache/Makefile.am                |    4 ++--
 plugins/colorspace-adobergb/Makefile.am  |    4 ++--
 plugins/colorspace-prophoto/Makefile.am  |    4 ++--
 plugins/colorspace-srgb/Makefile.am      |    4 ++--
 plugins/colorspace-transform/Makefile.am |    4 ++--
 plugins/crop/Makefile.am                 |    4 ++--
 plugins/dcp/Makefile.am                  |    4 ++--
 plugins/demosaic/Makefile.am             |    4 ++--
 plugins/denoise/Makefile.am              |    4 ++--
 plugins/exposure-mask/Makefile.am        |    4 ++--
 plugins/fuji-rotate/Makefile.am          |    4 ++--
 plugins/input-file/Makefile.am           |    4 ++--
 plugins/input-image16/Makefile.am        |    4 ++--
 plugins/lensfun/Makefile.am              |    4 ++--
 plugins/load-dcraw/Makefile.am           |    4 ++--
 plugins/load-gdk/Makefile.am             |    4 ++--
 plugins/load-rawspeed/Makefile.am        |    4 ++--
 plugins/meta-ciff/Makefile.am            |    4 ++--
 plugins/meta-exiv2/Makefile.am           |    4 ++--
 plugins/meta-mrw/Makefile.am             |    4 ++--
 plugins/meta-raf/Makefile.am             |    4 ++--
 plugins/meta-tiff/Makefile.am            |    4 ++--
 plugins/meta-x3f/Makefile.am             |    4 ++--
 plugins/output-facebook/Makefile.am      |    2 +-
 plugins/output-flickr/Makefile.am        |    2 +-
 plugins/output-jpegfile/Makefile.am      |    4 ++--
 plugins/output-picasa/Makefile.am        |    2 +-
 plugins/output-pngfile/Makefile.am       |    4 ++--
 plugins/output-tifffile/Makefile.am      |    4 ++--
 plugins/resample/Makefile.am             |    4 ++--
 plugins/rotate/Makefile.am               |    4 ++--
 34 files changed, 62 insertions(+), 61 deletions(-)

diff --git a/librawstudio/Makefile.am b/librawstudio/Makefile.am
index aab7893..73f2b15 100644
--- a/librawstudio/Makefile.am
+++ b/librawstudio/Makefile.am
@@ -7,6 +7,7 @@ AM_CPPFLAGS = \
 
 INCLUDES = \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
+	-DPACKAGE_LIBRARY_DIR=\""$(libdir)"\" \
 	-DPACKAGE_LOCALE_DIR=\""$(prefix)/$(DATADIRNAME)/locale"\" \
 	@PACKAGE_CFLAGS@
 
diff --git a/librawstudio/rawstudio-2.0.pc.in b/librawstudio/rawstudio-2.0.pc.in
index 4ccfc52..cd2c3d5 100644
--- a/librawstudio/rawstudio-2.0.pc.in
+++ b/librawstudio/rawstudio-2.0.pc.in
@@ -2,7 +2,7 @@ prefix=@prefix@
 exec_prefix=${prefix}
 libdir=@libdir@
 includedir=${prefix}/include
-plugindir=@prefix@/share/rawstudio/plugins
+plugindir=@prefix@/lib/rawstudio/plugins
 
 Name: @PACKAGE@
 Description: Rawstudio backend library
diff --git a/librawstudio/rs-plugin-manager.c b/librawstudio/rs-plugin-manager.c
index 9f14839..00b1a1f 100644
--- a/librawstudio/rs-plugin-manager.c
+++ b/librawstudio/rs-plugin-manager.c
@@ -38,7 +38,7 @@ rs_plugin_manager_load_all_plugins()
 
 	g_assert(g_module_supported());
 
-	plugin_directory = g_build_filename(PACKAGE_DATA_DIR, PACKAGE, "plugins", NULL);
+	plugin_directory = g_build_filename(PACKAGE_LIBRARY_DIR, PACKAGE, "plugins", NULL);
 	RS_DEBUG(PLUGINS, "Loading modules from %s", plugin_directory);
 
 	dir = g_dir_open(plugin_directory, 0, NULL);
diff --git a/plugins/cache/Makefile.am b/plugins/cache/Makefile.am
index 7c5f130..bc4839a 100644
--- a/plugins/cache/Makefile.am
+++ b/plugins/cache/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = cache.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 cache_la_LIBADD = @PACKAGE_LIBS@
 cache_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/colorspace-adobergb/Makefile.am b/plugins/colorspace-adobergb/Makefile.am
index 1e5d095..631523a 100644
--- a/plugins/colorspace-adobergb/Makefile.am
+++ b/plugins/colorspace-adobergb/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =	-Wall
 
@@ -13,7 +13,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = colorspace_adobergb.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 colorspace_adobergb_la_LIBADD = @PACKAGE_LIBS@
 colorspace_adobergb_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/colorspace-prophoto/Makefile.am b/plugins/colorspace-prophoto/Makefile.am
index 19bfce1..fff69e3 100644
--- a/plugins/colorspace-prophoto/Makefile.am
+++ b/plugins/colorspace-prophoto/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =	-Wall
 
@@ -13,7 +13,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = colorspace_prophoto.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 colorspace_prophoto_la_LIBADD = @PACKAGE_LIBS@
 colorspace_prophoto_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/colorspace-srgb/Makefile.am b/plugins/colorspace-srgb/Makefile.am
index dd34b4a..d21fdde 100644
--- a/plugins/colorspace-srgb/Makefile.am
+++ b/plugins/colorspace-srgb/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 INCLUDES = \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
@@ -9,7 +9,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = colorspace_srgb.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 colorspace_srgb_la_LIBADD = @PACKAGE_LIBS@
 colorspace_srgb_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/colorspace-transform/Makefile.am b/plugins/colorspace-transform/Makefile.am
index 5d600fe..a332486 100644
--- a/plugins/colorspace-transform/Makefile.am
+++ b/plugins/colorspace-transform/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall \
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = colorspace_transform.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 colorspace_transform_la_LIBADD = @PACKAGE_LIBS@ colorspace_transform_sse2.lo rs-cmm.lo colorspace_transform-c.lo
 colorspace_transform_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/crop/Makefile.am b/plugins/crop/Makefile.am
index b63e3c7..f7a98ca 100644
--- a/plugins/crop/Makefile.am
+++ b/plugins/crop/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = crop.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 crop_la_LIBADD = @PACKAGE_LIBS@
 crop_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/dcp/Makefile.am b/plugins/dcp/Makefile.am
index 837baa2..eca74ab 100644
--- a/plugins/dcp/Makefile.am
+++ b/plugins/dcp/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS = -O4 -Wall -ffast-math -ffinite-math-only \
 	-funsafe-math-optimizations -funroll-loops
@@ -12,7 +12,7 @@ INCLUDES = -DPACKAGE_DATA_DIR=\""$(datadir)"\" \
 
 lib_LTLIBRARIES = dcp.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 dcp_la_LIBADD = @PACKAGE_LIBS@ adobe-camera-raw-tone.lo dcp-sse2.lo dcp-sse4.lo dcp-c.lo
 dcp_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/demosaic/Makefile.am b/plugins/demosaic/Makefile.am
index 82795f2..00c3ace 100644
--- a/plugins/demosaic/Makefile.am
+++ b/plugins/demosaic/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = demosaic.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 demosaic_la_LIBADD = @PACKAGE_LIBS@
 demosaic_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/denoise/Makefile.am b/plugins/denoise/Makefile.am
index d71b846..39396eb 100644
--- a/plugins/denoise/Makefile.am
+++ b/plugins/denoise/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall \
@@ -16,7 +16,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = denoise.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 denoise_la_LIBADD = @PACKAGE_LIBS@
 denoise_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/exposure-mask/Makefile.am b/plugins/exposure-mask/Makefile.am
index 41173cb..5169b10 100644
--- a/plugins/exposure-mask/Makefile.am
+++ b/plugins/exposure-mask/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = exposure_mask.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 exposure_mask_la_LIBADD = @PACKAGE_LIBS@
 exposure_mask_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/fuji-rotate/Makefile.am b/plugins/fuji-rotate/Makefile.am
index 38588ef..fa96cf0 100644
--- a/plugins/fuji-rotate/Makefile.am
+++ b/plugins/fuji-rotate/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = fuji_rotate.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 fuji_rotate_la_LIBADD = @PACKAGE_LIBS@
 fuji_rotate_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/input-file/Makefile.am b/plugins/input-file/Makefile.am
index cb3d0a3..77a50bb 100644
--- a/plugins/input-file/Makefile.am
+++ b/plugins/input-file/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = input_file.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 input_file_la_LIBADD = @PACKAGE_LIBS@
 input_file_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/input-image16/Makefile.am b/plugins/input-image16/Makefile.am
index 19a132c..87f31f7 100644
--- a/plugins/input-image16/Makefile.am
+++ b/plugins/input-image16/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = input_image16.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 input_image16_la_LIBADD = @PACKAGE_LIBS@
 input_image16_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/lensfun/Makefile.am b/plugins/lensfun/Makefile.am
index f2e7425..fa3ce01 100644
--- a/plugins/lensfun/Makefile.am
+++ b/plugins/lensfun/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -16,7 +16,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = lensfun.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 lensfun_la_LIBADD = @PACKAGE_LIBS@ lensfun-sse2.lo lensfun-c.lo
 lensfun_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/load-dcraw/Makefile.am b/plugins/load-dcraw/Makefile.am
index df97c7d..c6b7e45 100644
--- a/plugins/load-dcraw/Makefile.am
+++ b/plugins/load-dcraw/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -18,7 +18,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = load_dcraw.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 load_dcraw_la_LIBADD = @PACKAGE_LIBS@
 load_dcraw_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/load-gdk/Makefile.am b/plugins/load-gdk/Makefile.am
index 694ddf3..a82e3f2 100644
--- a/plugins/load-gdk/Makefile.am
+++ b/plugins/load-gdk/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = load_gdk.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 load_gdk_la_LIBADD = @PACKAGE_LIBS@
 load_gdk_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/load-rawspeed/Makefile.am b/plugins/load-rawspeed/Makefile.am
index edf8a18..75b3c97 100644
--- a/plugins/load-rawspeed/Makefile.am
+++ b/plugins/load-rawspeed/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -16,7 +16,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = load_rawspeed.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 load_rawspeed_la_LIBADD = @PACKAGE_LIBS@
 load_rawspeed_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/meta-ciff/Makefile.am b/plugins/meta-ciff/Makefile.am
index 7a63115..5095f62 100644
--- a/plugins/meta-ciff/Makefile.am
+++ b/plugins/meta-ciff/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = meta_ciff.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 meta_ciff_la_LIBADD = @PACKAGE_LIBS@
 meta_ciff_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/meta-exiv2/Makefile.am b/plugins/meta-exiv2/Makefile.am
index 6effbea..5f9e968 100644
--- a/plugins/meta-exiv2/Makefile.am
+++ b/plugins/meta-exiv2/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = meta_exiv2.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 meta_exiv2_la_LIBADD = @PACKAGE_LIBS@
 meta_exiv2_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/meta-mrw/Makefile.am b/plugins/meta-mrw/Makefile.am
index 3d3098a..be936fa 100644
--- a/plugins/meta-mrw/Makefile.am
+++ b/plugins/meta-mrw/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = meta_mrw.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 meta_mrw_la_LIBADD = @PACKAGE_LIBS@
 meta_mrw_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/meta-raf/Makefile.am b/plugins/meta-raf/Makefile.am
index 9deb195..9de212a 100644
--- a/plugins/meta-raf/Makefile.am
+++ b/plugins/meta-raf/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = meta_raf.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 meta_raf_la_LIBADD = @PACKAGE_LIBS@
 meta_raf_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/meta-tiff/Makefile.am b/plugins/meta-tiff/Makefile.am
index 03771ac..fafe941 100644
--- a/plugins/meta-tiff/Makefile.am
+++ b/plugins/meta-tiff/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = meta_tiff.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 meta_tiff_la_LIBADD = @PACKAGE_LIBS@
 meta_tiff_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/meta-x3f/Makefile.am b/plugins/meta-x3f/Makefile.am
index 07d70d3..a98d711 100644
--- a/plugins/meta-x3f/Makefile.am
+++ b/plugins/meta-x3f/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = meta_x3f.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 meta_x3f_la_LIBADD = @PACKAGE_LIBS@
 meta_x3f_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/output-facebook/Makefile.am b/plugins/output-facebook/Makefile.am
index eaca619..3d2a18a 100644
--- a/plugins/output-facebook/Makefile.am
+++ b/plugins/output-facebook/Makefile.am
@@ -16,7 +16,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = output_facebook.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 output_facebook_la_LIBADD = @PACKAGE_LIBS@ 
 output_facebook_la_LDFLAGS = -module -avoid-version -L/usr/lib -lcurl
diff --git a/plugins/output-flickr/Makefile.am b/plugins/output-flickr/Makefile.am
index 06cb90c..ba89b10 100644
--- a/plugins/output-flickr/Makefile.am
+++ b/plugins/output-flickr/Makefile.am
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = output_flickr.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 output_flickr_la_LIBADD = @PACKAGE_LIBS@ 
 output_flickr_la_LDFLAGS = -module -avoid-version -L/usr/lib -lflickcurl
diff --git a/plugins/output-jpegfile/Makefile.am b/plugins/output-jpegfile/Makefile.am
index 9f95681..ce3aa6e 100644
--- a/plugins/output-jpegfile/Makefile.am
+++ b/plugins/output-jpegfile/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = output_jpegfile.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 output_jpegfile_la_LIBADD = @PACKAGE_LIBS@
 output_jpegfile_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/output-picasa/Makefile.am b/plugins/output-picasa/Makefile.am
index 1c5e13a..9e7d828 100644
--- a/plugins/output-picasa/Makefile.am
+++ b/plugins/output-picasa/Makefile.am
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = output_picasa.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 output_picasa_la_LIBADD = @PACKAGE_LIBS@ 
 output_picasa_la_LDFLAGS = -module -avoid-version -L/usr/lib
diff --git a/plugins/output-pngfile/Makefile.am b/plugins/output-pngfile/Makefile.am
index 56c2947..63226af 100644
--- a/plugins/output-pngfile/Makefile.am
+++ b/plugins/output-pngfile/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = output_pngfile.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 output_pngfile_la_LIBADD = @PACKAGE_LIBS@
 output_pngfile_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/output-tifffile/Makefile.am b/plugins/output-tifffile/Makefile.am
index 86a8e94..a6bfa76 100644
--- a/plugins/output-tifffile/Makefile.am
+++ b/plugins/output-tifffile/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = output_tifffile.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 output_tifffile_la_LIBADD = @PACKAGE_LIBS@
 output_tifffile_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/resample/Makefile.am b/plugins/resample/Makefile.am
index 230da37..118f76f 100644
--- a/plugins/resample/Makefile.am
+++ b/plugins/resample/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = resample.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 resample_la_LIBADD = @PACKAGE_LIBS@ resample-sse2.lo resample-c.lo
 resample_la_LDFLAGS = -module -avoid-version
diff --git a/plugins/rotate/Makefile.am b/plugins/rotate/Makefile.am
index 26cade9..a106faf 100644
--- a/plugins/rotate/Makefile.am
+++ b/plugins/rotate/Makefile.am
@@ -1,4 +1,4 @@
-plugindir = $(libdir)/rawstudio/plugins
+plugindir := /usr/lib/rawstudio/plugins
 
 AM_CFLAGS =\
 	-Wall\
@@ -15,7 +15,7 @@ INCLUDES = \
 
 lib_LTLIBRARIES = rotate.la
 
-libdir = $(datadir)/rawstudio/plugins/
+libdir := /usr/lib/rawstudio/plugins/
 
 rotate_la_LIBADD = @PACKAGE_LIBS@
 rotate_la_LDFLAGS = -module -avoid-version
-- 
